recipes/image-meta-tag/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = "0.5.0" %}
+
+package:
+    name: image-meta-tag
+    version: {{ version }}
+
+source:
+    git_url: https://github.com/SciTools-incubator/image-meta-tag
+    git_tag: {{ version }}
+    sha1: f0b204bdb789cb8c8efb5cf9b30f9677684f0728
+
+build:
+    number: 0
+
+requirements:
+    build:
+        - python >=2.7,<3
+        - setuptools
+    run:
+        - python >=2.7,<3
+        - matplotlib
+        - numpy
+        - pillow
+
+test:
+    imports:
+        - ImageMetaTag
+
+about:
+    license: BSD3
+    summary: 'Image metadata tagging, database and presentation'
+    home: https://github.com/SciTools-incubator/image-meta-tag
+
+extra:
+  recipe-maintainers:
+    - marqh
+    - malcolmbrooks
```

Comment:
@malcolmbrooks - are you ok being added as a co-maintainer of the ImageMetaTag recipe on conda-forge? You were listed by @marqh in the PR.